### ISSUE TITLE
Got an Error of a depreciated package 

### ISSUE THREAD

**Issue by sheggietyn** [roto93/react-native-neat-date-picker#7]
EventEmitter.removeListerner('didUpdateDimensions,') method has been deprecated, please instead use remove() on the subscription returned by EventEmitter.addListener

that was the error i got 

and secondly the output date shows only Day  it didn't show  the full date

**Comment by roto93** [comment]
Hi, Are you using Expo? 
I guess thehe EventEmitter error is because I didn't use the latest version of react-native-modal for my date picker. (I saw this [PR](https://github.com/react-native-modal/react-native-modal/pull/594#issuecomment-904510835))
Because of [this reason](https://github.com/roto93/react-native-neat-date-picker/issues/5#issuecomment-993851563), I have to downgrade the react-native-modal in order to use it. I'm still seeking a good way to handle the version problem. Otherwise we have to wait for Expo SDK includes RN > 0.65. At that time, the latest rn-modal version should work.

For the second question, yes the output is a javascript Date object. This decision was because everyone can get their information using Date prototype method. But some people like you ask to get the full date string so I am now considering adding this feature. Thank you for your suggestion. 👍

**Comment by roto93** [comment]
I have updated a new feature in version 1.2.0. Now you can get the full date easily.
And I changed the way to use onConfirm prop. Please check README to see how to use it.

**Comment by sheggietyn** [comment]
thanks for the update i will check it out now

**Comment by sheggietyn** [comment]
i still got this error when i tried again 

EventEmitter.removeListerner('didUpdateDimensions,') method has been deprecated, please instead use remove() on the subscription returned by EventEmitter.addListener

**Comment by roto93** [comment]
I can expect that because I only dealt with you second question.
I've tried the first one but I cannot reproduce it. 
Is it just a warning (i.e. you can still use it) or an Error that makes the date picker totally unavailable?
Though I believe it is the dependency that causing this error, could you provide more information or a minimal reproduction?

**Comment by arkanos** [comment]
Hi @roto93,

After some investigation, I have found that this warning was caused by **react-native-modal** package, at least on my project.

I have tried to reproduce it within another project with a clean expo installation, with latest version (13.0.1) and seems everything works as expected. No sign of warning at all after a while.

Looks like the library was using a deprecated method and was fixed in the latest updates.

Hopes this helps a bit!
